test/test_nn.py

```diff
@@ -241,6 +241,17 @@ def _get_parameters(self, module):
             d_params.append(p.grad.data)
         return params, d_params
 
+    def _assert_grad_and_gradgradchecks(self, apply_fn, inputs):
```

Comment:
Just a bit: Maybe let make it look more like `assertEqual` I.e. Use camel case for all asserts